### ISSUE TITLE
Zabbix error - Undefined 500 Internal Server Error

### ISSUE THREAD

**Issue by pablosouza10** [nuxsmin/sysMonDash#19]
I using Zabbix 3.0, in previous release work fine, but the last version show error 500.

Any help?
![errorr](https://cloud.githubusercontent.com/assets/17860003/18713431/b19ae668-7fe8-11e6-9026-e167e5fa24a9.PNG)

**Comment by nuxsmin** [comment]
Is there any error at the webserver event log?

**Comment by pablosouza10** [comment]
Hi nuxsmin,

the error log apache:

[Thu Sep 22 09:49:33 2016] [error] [client 192.168.XX.21] PHP Fatal error:  Call to undefined function Exts\Zabbix\V225\posix_getuid() in /var/www/html/sysMonDash/inc/Exts/Zabbix/V225/ZabbixApiAbstract.class.php on line 472, referer: http://10.XX.XX.51/sysMonDash/

This 472 line is :  

 if($tokenCacheDir && array_key_exists('user', $params) && is_dir($tokenCacheDir))
            $tokenCacheFile = $tokenCacheDir.'/.zabbixapi-token-'.md5($params['user'].'|'.posix_getuid());

PS: I using in the same server sysMonDash old version in production.

same tip?

**Comment by nuxsmin** [comment]
Are you running SMD on windows?. It seems that posix_getuid function is missing.

**Comment by nuxsmin** [comment]
I can code some hack to get it working.

**Comment by pablosouza10** [comment]
No, I'm using on Centos.

I guess the problem is the .zabbixapi-token-e3afed0047b08059d0fada10f400c1e5 int the tmp dir was create from old version and conflict with new version.

**Comment by pablosouza10** [comment]
Hi,

"Disable caching of auth tokens

If you want to disable the caching, set $tokenCacheDir to an empty value ''.

Work for me.

**Comment by nuxsmin** [comment]
Right, but I wouldn't disable auth caching forever, because it could produce a slightly server overhead.

Let me find out a solution for this issue.

**Comment by pablosouza10** [comment]
Ok. great

Thanks, waiting for solutions.
